sources/packages/web/src/components/layouts/Institution/sidebar/HomeSideBar.vue

```diff
@@ -2,20 +2,26 @@
   <Menu :model="items" />
 </template>
 <script lang="ts">
-import Menu from "primevue/menu";
-import { ref } from "vue";
+import Menu from "primevue/panelmenu";
+import { useRouter } from "vue-router";
+import { UserService } from "../../../../services/UserService";
+import { ref, onMounted } from "vue";
+import { InstitutionRoutesConst } from "@/constants/routes/RouteConstants";
 
 interface MenuModel {
   label: string;
   icon?: string;
   command?: () => void;
+  items?: MenuModel[];
 }
 
 export default {
   components: {
     Menu,
   },
   setup() {
+    const router = useRouter();
+    const subItems = ref<MenuModel[]>([]);
```

Comment:
It seems that this variable is not used, can you check and remove it if possible?

Reply:
Done